src/winforms/toga_winforms/widgets/imageview.py

```diff
@@ -0,0 +1,34 @@
+from toga_winforms.libs import *
+
+from .base import Widget
+
+
+class ImageView(Widget):
+
+    def create(self):
+        self.native = WinForms.PictureBox()
+        self.native.interface = self.interface
+        self.native.BorderStyle = WinForms.BorderStyle.Fixed3D
+
+    def get_image(self):
+        return self.native.image
+
+    def set_image(self, image):
+        if image:
+            # Workaround for loading image from url
+            if isinstance(image._impl.native, str):
+                self.native.Load(image._impl.native)
```

Comment:
Out of interest - is this a blocking load? If the URL takes a while to resolve and download, does the rest of the GUI freeze?

Reply:
When a 40mb image url is added to the example app, it takes a while to open the app. So, I guess, it's blocking? I couldn't implement on_selection...